tests/test_api.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import numpy as np
+from libsvmdata import fetch_dataset, print_supported_datasets
+
+
+# TODO : add other datasets to test ?
+def test_replace():
+    X_first, y_first = fetch_dataset("iris", replace=True)
+    X_second, y_second = fetch_dataset("iris")
+    assert np.all(X_first == X_second)
```

Comment:
use np.testing.assert_equal: it will give a more useful error message in case of failure